packages/plugin-ext/src/hosted/node/scanners/scanner-theia.ts

```diff
@@ -60,6 +60,7 @@ import { PreferenceSchema, PreferenceSchemaProperties } from '@theia/core/lib/co
 import { RecursivePartial } from '@theia/core/lib/common/types';
 import { ProblemMatcherContribution, ProblemPatternContribution, TaskDefinition } from '@theia/task/lib/common/task-protocol';
 import { ColorDefinition } from '@theia/core/lib/browser/color-registry';
+import { ResourceLabelFormatter } from '@theia/core/lib/browser';
```

Comment:
the same here, we import here browser code in node

Reply:
moved `ResourceLabelFormatter` to common in core